src/Cassandra/Mapping/ICqlQueryAsyncClient.cs

```diff
@@ -25,20 +25,43 @@ namespace Cassandra.Mapping
     public interface ICqlQueryAsyncClient
     {
         /// <summary>
-        /// Gets a list of all T from Cassandra.
+        /// Gets a list of all T from Cassandra. Loading new pages when enumerating the result may block the thread. For that reason,
+        /// FetchAsAsyncEnumerable should be preferred.
```

Comment:
`FetchAsAsyncEnumerable should be preferred if the .NET version supports it`

Reply:
Done